Update set-up-snowflake-native-app.md (#5658)
fishtown-analytics/docs.getdbt.com: website/docs/docs/cloud-integrations/set-up-snowflake-native-app.md

- You have **ACCOUNTADMIN** access in Snowflake.
- Your Snowflake account must have access to the Native App/SPCS integration (PrPr until Summit) and NA/SPCS configurations (PuPr at end of June). If you're unsure, please check with your Snowflake account manager.
- The Snowflake account must be in an AWS Region or Azure region.

> - You have access to Snowflake Cortex via your Snowflake permissions as well as the [region](https://docs.snowflake.com/en/user-guide/snowflake-cortex/llm-functions#availability). If you do not, dbt Ask will be broken. 

Author: ```suggestion
- You have access to Snowflake Cortex via your Snowflake permissions as well as the [region](https://docs.snowflake.com/en/user-guide/snowflake-cortex/llm-functions#availability). If you do not, `Ask dbt` will be broken. 
```

> - You have access to Snowflake Cortex via your Snowflake permissions as well as the [region](https://docs.snowflake.com/en/user-guide/snowflake-cortex/llm-functions#availability). If you do not, dbt Ask will be broken. 

Reviewer: ```suggestion
- You have access to Snowflake Cortex through your Snowflake permissions and the [correct region](https://docs.snowflake.com/en/user-guide/snowflake-cortex/llm-functions#availability). Without this, dbt Ask will not work.
```

> - You have access to Snowflake Cortex through your Snowflake permissions and the [correct region](https://docs.snowflake.com/en/user-guide/snowflake-cortex/llm-functions#availability). Without this, dbt Ask will not work.

Reviewer: ```suggestion
- You have access to Snowflake Cortex through your Snowflake permissions and the [correct Snowflake Cortex region](https://docs.snowflake.com/en/user-guide/snowflake-cortex/llm-functions#availability). Without this, dbt Ask will not work.
```

> - You have access to Snowflake Cortex through your Snowflake permissions and the [correct Snowflake Cortex region](https://docs.snowflake.com/en/user-guide/snowflake-cortex/llm-functions#availability). Without this, dbt Ask will not work.

Author: ```suggestion
- You have access to Snowflake Cortex through your Snowflake permissions and [Snowflake Cortex is available in your region](https://docs.snowflake.com/en/user-guide/snowflake-cortex/llm-functions#availability). Without this, Ask dbt will not work.
```
